### ISSUE TITLE
Fonts should be defined in relative sizes

### ISSUE THREAD

**Issue by elizabethsjudd** [Explorys/ink#18]
In the Watson Theme the fonts are sized with pixels instead of a relative unit.  For accessibility guidelines font sizes should be converted to em/rems.
